File: src/sdk-reference/go/1/document/count/snippets/count.go
count, err := kuzzle.Document.Count("nyc-open-data", "yellow-taxi", json.RawMessage(`{"query": {"match": {"licence": "valid"}}}`), nil)

if err != nil {
  log.Fatal(err)
} else {
  fmt.Printf("Found %d documents matching licence:valid\n", count)
}

File: src/sdk-reference/cpp/1/document/count/snippets/count.cpp
try {
  int count = kuzzle->document->count("nyc-open-data", "yellow-taxi", R"({
    "query": {
      "match": {
        "license": "valid"
      }
    }
  })");

  std::cout << "Found " << count << " documents matching license:valid" << std::endl;
} catch (kuzzleio::KuzzleException e) {
  std::cerr << e.what() << std::endl;
}
